### PR TITLE
chore: bump `next-moc`, update changelog in prep for 0.6.21

### PR DESCRIPTION
**Pull request by crusso**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
   * Fix: insert critical overflow checks preventing rare heap corruptions
-    in out-of-memory allocation and stable variable serialization (#3077) 
-  * Implement support for 128-bit Cycles-API (#3042)
+    in out-of-memory allocation and stable variable serialization (#3077).
+  * Implement support for 128-bit Cycles-API (#3042).
 
 * motoko-base
 
-  * `Principal.fromBlob` added (#331)
+  * `ExperimentalInternetComputer` library, exposing low-level, binary `call` function (a.k.a `raw calls`) (#334, Motoko #3806).
```
Comment (ggreif) [inline Changelog.md:14]:
```suggestion
  * `ExperimentalInternetComputer` library, exposing low-level, binary `call` function (a.k.a. `raw calls`) (#334, Motoko #3806).
```

File: Changelog.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+  * `Principal.fromBlob` added (#331).
```
Comment (ggreif) [inline Changelog.md:15]:
```suggestion
  * `Principal.fromBlob` added (dfinity/motoko-base#331).
```

**Comment by crusso** [comment]
(#ggeif In future, I think we should avoid adding the release date until we've actually released - makes it clearer and avoid slippage. For second, I thought you'd released 0.6.21 already and went looking for a tag)

**Comment by ggreif** [comment]
I suggest adding the `dfinity/motoko-base`-prefix to `(#328)` too. The release has sailed, but I can change it in the release page manually.

**Comment by ggreif** [comment]
Right, we should write `(future)`.

**Review by ggreif — APPROVED** [review]

**Pull request merged**